docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,841 @@
+    "\n",
+    "# Construct the optimizer\n",
+    "qrao = QuantumRandomAccessOptimizer(min_eigen_solver=vqe, rounding_scheme=semidterministic_rounding)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we move forward with solving the problem by invoking the `solve()` method. It's important to note that when calling `solve()`, we pass the `problem` itself as an argument. Although we previously used `encode()` in `QuantumRandomAccessEncoding` to provide a clear understanding of the flow, `solve(problem)` automatically encodes the problem internally using `QuantumRandomAccessEncoding`. This provides a streamlined and simplified workflow that eliminates the need for explicit encoding steps.\n",
+    "\n",
+    "The result is provides us as a `QuantumRandomAccessOptimizationResult`.\n",
+    "The `x` contains the binary values representing the best solution found, while the `fval` contains the corresponding objective value.\n",
+    "\n",
+    "The `relaxed_fval` provides the expectation value of the relaxed Hamiltonian, adjusted to be in the units of the original optimization problem. For maximization problems, the best possible relaxed function value will always be greater than or equal to the best possible function value of the original problem. In practice, this often holds true for the best found value and best found function value as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The objective function value: 5.0\n",
+      "x: [0 1 1 0 1 0]\n",
+      "relaxed function value: 8.999999983259919\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Solve the optimization problem\n",
+    "results = qrao.solve(problem)\n",
+    "\n",
+    "print(\n",
+    "    f\"The objective function value: {results.fval}\\n\"\n",
+    "    f\"x: {results.x}\\n\"\n",
+    "    f\"relaxed function value: {-1 * results.relaxed_fval}\\n\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Interpret the solution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the context of [max-cut](https://en.wikipedia.org/wiki/Maximum_cut), the result's \"optimal value\" tells us which subset each node belongs to given the partition found by the optimizer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The obtained solution places a partition between nodes [0 3 5] and nodes [1 2 4].\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "print(\n",
+    "    f\"The obtained solution places a partition between nodes {np.where(results.x == 0)[0]} \"\n",
```

Comment:
Why not use `Maxcut.interpret`? It generates partitions from the result.
https://qiskit.org/ecosystem/optimization/stubs/qiskit_optimization.applications.Maxcut.interpret.html#qiskit_optimization.applications.Maxcut.interpret